handler.py

```diff
@@ -14,38 +16,120 @@
 SQS_CLIENT = boto3.client('sqs')
 
 
-def runScheduledObservatoryScan(event, context):
+def addPortScanToQueue(event, context):
```

Comment:
Because port scans are less bounded than  say Observatory scans, should we consider adding some form of authentication, like a static token to ensure  that these  requests are properly authenticated before being kicked off?  I'm thinking that our abuse potential rises here and we should make note of that (possibly some action).

Reply:
Yes good point. I will implement something for the on-demand port-scan ingestion point